src/components/task-teacher-review/crumb-mixin.cjsx

```diff
@@ -66,16 +66,29 @@ module.exports =
       _.each crumb.data, (data) ->
         data.key = crumb.key
     else
-      crumb.data.key = crumb.key
+      crumb.data.key = index
 
 
   _getCrumbsForHomework: (stats) ->
     exercises = @_getExercisesFromStats(stats)
     crumbs = _.chain(exercises)
-      .map(@_makeCrumb)
-      .each(@_indexCrumb)
+      .map((data) =>
+        {questions} = JSON.parse(data.content)
+        stepCrumbs = []
```

Comment:
yeah that's true. If it wasn't JSON we'd have exceptions.  Here's what I see though:
![screen shot 2016-05-27 at 1 31 21 pm](https://cloud.githubusercontent.com/assets/79566/15617616/5816059e-240f-11e6-8256-88f9f565ae7e.png)